### PR TITLE
 update time css

### PR DESCRIPTION
**Pull request by Alex-Beep-Cao**
**Issue:**
No Issue

**Solution:**
Just update timer css

**Risk:**
No risk 

**Reviewed By:**
Jason Jason

### CODE REVIEW

File: frontend/src/components/timer/timer.module.css [hunk 1]
```diff
@@ -5,11 +5,13 @@
 
 .timer {
```
Comment (jekb2019) [inline frontend/src/components/timer/timer.module.css:6]: It seems like separate styles for ".red" and ".blue" class is not defined in the CSS. How are you planning to implement this feature? - Let's have a discussion whether separate background colour is necessary for Timer component. According to our Figma, it seems like the timer colour scheme is always black/white. 

If we think adding red/blue will make our app cooler, we can definitely try it out!

**Review by jekb2019 — COMMENTED** [review]
The Timer circle is nicely styled and is looking fantastic 👍 

At the moment, Timer component stores the "sec" state which represents the seconds past.

Timer and TimerBar are children of GamePage component. They both need access to the "sec" state as their functionalities will depend on it. Therefore it will make more sense to store the "sec" state inside the GamePage component and pass it as a prop to both Timer and TimerBar.

Plz check out my other comments about the CSS as well.

File: frontend/src/components/timer/timer.module.css (hunk 1 above)
Comment (Alex-Beep-Cao) [reply frontend/src/components/timer/timer.module.css:6]: oh i forget to delete red and blue class, I used them to change the background color in my first timer version. If we want to keep same as our Figma  we can just delete red and blue class. And about time bar, i am waiting for Jason (maybe) to finish then we can merge together

**Review by jzha411 — APPROVED** [review]

**Review by jekb2019 — APPROVED** [review]
LGTM!

**Pull request merged**